src/tiles/text-snippet-block.ts

```diff
@@ -0,0 +1,116 @@
+import { css, CSSResultGroup, html, LitElement, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { map } from 'lit/directives/map.js';
+import { join } from 'lit/directives/join.js';
+import DOMPurify from 'dompurify';
+
+@customElement('text-snippet-block')
+export class TextSnippetBlock extends LitElement {
+  // The default snippets value here is just for testing's sake; will be removed.
+  @property({ type: Array }) snippets?: string[] = [
```

Comment:
```suggestion
  @property({ type: Array }) snippets?: string[] = [];
```
hmmm, what's a good way to toggle a set of results with FTS snippets from `<app-root>` so we may remove this placeholder?

light digging, looks like we can probably create an FTS SearchService stub that returns a hard coded set of results.  So that when toggling a checkbox/radio button on demo controls, we can just display the results set with `snippet` included in - like in model update.

what do you think?

Reply:
Thanks Isa! Yeah, I was grappling with what might be a reasonable way to apply the test snippets for the demo app. This approach sounds good to me.

Reply:
The snippets can now be toggled from the demo controls (by wrapping the search-service to add hard-coded snippets). The placeholders are no longer present.